storage/client.go

```diff
@@ -308,6 +308,15 @@ func (c Client) buildCanonicalizedResourceTable(uri string) (string, error) {
 		cr += u.EscapedPath()
 	}
 
+	params, err := url.ParseQuery(u.RawQuery)
+
+	// search for "comp" parameter, if exists then add it to canonicalizedresource
+	for key := range params {
```

Comment:
The only way I figured out what to do was to disassemble the dotnet binaries (then check github... duh)....    see https://github.com/Azure/azure-storage-net/blob/68c3ee55a3a6f62a0159cea58005d3fe027312a8/Lib/Common/Core/Util/AuthenticationUtility.cs#L313

After I did the same, everything sharedkeylite started working for me.